api/v1alpha1/stardoguser_types.go

```diff
@@ -25,19 +26,42 @@ import (
 
 // StardogUserSpec defines the desired state of StardogUser
 type StardogUserSpec struct {
-	// INSERT ADDITIONAL SPEC FIELDS - desired state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	// StardogInstance describes a Stardog instance
+	StardogInstance string `json:"stardogInstance,omitempty"`
```

Comment:
The `ref` suffix better indicates that this is a reference to StardogInstance, not the URL itself
```suggestion
	// StardogInstanceRef describes a Stardog instance
	StardogInstanceRef string `json:"stardogInstanceRef,omitempty"`
```